packages/core/src/components/forms/numericInput.tsx

```diff
@@ -236,8 +246,11 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
             const decrementButton = this.renderButton(
                 NumericInput.DECREMENT_KEY, NumericInput.DECREMENT_ICON_NAME, this.handleDecrementButtonClick);
 
+            const buttonGroupClasses = classNames(Classes.BUTTON_GROUP, Classes.VERTICAL, Classes.FIXED, {
+                [Classes.LARGE]: large,
```

Comment:
I see. Something else to note is that `pt-input-group` gets large styling for free, but only when `pt-large` is applied directly to it. Seems like the choices are:

**Option 1**
```
.pt-numeric-input.pt-large
   .pt-input-group // need to copy a bunch of large input group styles here (maybe via a mixin)
   .pt-button-group .pt-button // need to scope within .pt-button-group b/c the input group may contain buttons too
```

or

**Option 2**
```
.pt-numeric-input.pt-large
   .pt-input-group.pt-large // get large input-group styles for free (from _input-group.scss)
   .pt-button-group .pt-button // get large button styles for free (from _button.scss) due to pt-large ancestor
```

or

**Option 3** *(currently implemented)*
```
.pt-numeric-input
   .pt-input-group.pt-large // get large input-group styles for free
   .pt-button-group.pt-large .pt-button // get large button styles for free
```

Which feels best?